docs/performanceprofile/configuration_hotfixes.md

```diff
@@ -8,6 +8,57 @@ Instructions for editing (add/remove/change) kernel arguments,sysfs,proc paramet
 
 Default tunings are applied with the [openshift-performance](../../assets/performanceprofile/tuned/openshift-node-performance) base profile, it is the base for creating a Tuned CR that would be detected by the Node Tuning Operator and finally be executed by [tuned](https://github.com/redhat-performance/tuned).
 
+## RPS settings
+
+The default RPS settings for a performance profile are to set the RPS mask as the [reserved CPUs](performance_profile.md#cpu),\
+on the host level for all network devices excluding virtual(veth) devices and physical devices(pci)\
+and on the container level for all virtual network devices(veth).
+### RPS and workload hints
+
+When the realtime workload hint is explicitly disabled there is no need for any RPS settings to be applied since it is relevant only for the realtime use case.\
+The following will result in no RPS settings applied on the cluster at all:
+
+```yaml
+performance_profile.yaml
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: example-performanceprofile
+spec:
+  workloadHints:
+    realTime: false
+```
+
+In special cases where there is a need to explicitly specify the realtime workload hint as false but keep the RPS settings,
+an override annotation `performance.openshift.io/enable-rps` could be added to the performance profile that will keep the default [RPS settings](#rps-settings):
+
+```yaml
+performance_profile.yaml
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: example-performanceprofile
+  annotations:
+     performance.openshift.io/enable-rps: "true"  
+spec:
+  workloadHints:
+    realTime: false
+```
+
+### Enable RPS on physical devices annotation
+
+In case there is a need to set RPS mask for physical(pci) devices as well on the host side an override annotation `performance.openshift.io/enable-physical-dev-rps` to the default [RPS settings](#rps-settings) could be added to the performance profile:
+
+```yaml
+performance_profile.yaml
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: example-performanceprofile
+  annotations:
+     performance.openshift.io/enable-physical-dev-rps: "true"  
```

Comment:
This probably does not make any sense without the annotation above.

Reply:
how come ?  if realtime workload hint is not false then it can be added. 
in case realtime workload = false then we must have the annotation above inorder for it to make sense

Reply:
Well if I am reading the code right, then this annotation will not even be checked if realtime WH is false and the other annotation is missing. But the example does not say so.

Reply:
with realtime WH it will only check the combination with `performance.openshift.io/enable-rps`
I can specify that it will not check also `performance.openshift.io/enable-physical-dev-rps`